apt.txt

```diff
@@ -6,3 +6,5 @@ zlib1g-dev
 net-tools
 libpq-dev
 postgresql-client
+build-essential
+libcairo2-dev
```

Comment:
`python:3.10.16-slim` doesn't include `wget` so the `web` container won't start. It does a health check to see if the `watch` and `refine` containers are up using a script that depends on it - [we got that script from elsewhere](https://github.com/mitodl/mitxonline/blob/main/scripts/wait-for.sh) so it would probably be easiest to just add `wget`.